### PR TITLE
Android guidelines - Support for content language

### PR DESCRIPTION
**Pull request by terezijaumiljanovic**
Add guidelines for support for content language [#31](https://github.com/infinum/accessibility-mobile-standards/issues/31)

### CODE REVIEW

File: docs/guidelines/platforms/android/guideline_understandable_android.md [hunk 1]
```diff
@@ -2,13 +2,15 @@
...
 
-### Language of the App
+### Language of Page (WCAG 3.1.1 -Level A)
 
-*This guideline covers point 3.1.1 Language of Page - Level A of the WCAG standard.*
+This guideline covers point [3.1.1 Language of Page - Level A](https://www.w3.org/WAI/WCAG22/quickref/#language-of-page) of the WCAG standard.
+
+The default human language of an App can be programmatically determined.
 
 :white_check_mark: **Success criteria**
```
Comment (Aksi0m) [inline docs/guidelines/platforms/android/guideline_understandable_android.md:15]: Does it make sense to you that we somehow include our library [Localian](https://github.com/infinum/android-localian) in this section, as some kind of helper for handling locales?

**Review by Aksi0m — APPROVED** [review]
Seems good to me!
When I was reading this one idea came up to my mind. Wondering what you think -Should we maybe make [Localian](https://github.com/infinum/android-localian) a bit more accessibility focused in order to make our lives easier during development? Did not check the library in detail so it might be nonsense what I think. Just putting my mind out here :D

File: docs/guidelines/platforms/android/guideline_understandable_android.md [hunk 2]
```diff
@@ -22,10 +24,59 @@ It is also important to support localization because of accessibility services.
...
+val text = "Arc de Triomphe"
+val spannableString = SpannableString(text)
+
+// Set the language for accessibility
+val locale = Locale("fr", "FR")
+spannableString.setSpan(LocaleSpan(locale), 0, text.length, Spannable.SPAN_EXCLUSIVE_EXCLUSIVE)
+
+// Assign the spannable string to a TextView
+textView.text = spannableString
+```
```
Comment (zavrskina) [inline docs/guidelines/platforms/android/guideline_understandable_android.md:78]: Can we add Compose example as well? I believe you need to build an annotated String to achieve the same behavior, right?

File: docs/guidelines/platforms/android/guideline_understandable_android.md (hunk 1 above)
Comment (zavrskina) [reply docs/guidelines/platforms/android/guideline_understandable_android.md:15]: I believe we could mention it as an example of  library that can simplify this logic 👍

**Review by zavrskina — APPROVED** [review]

**Comment by terezijaumiljanovic** [comment]
> Should we maybe make [Localian](https://github.com/infinum/android-localian) a bit more accessibility focused in order to make our lives easier during development

Hmm, I'm not very familiar with Localian since I haven't used it. Perhaps we could reach out to the library's owner to discuss the potential for accessibility improvements and see if they can plan tasks for the future. 😄

**Pull request merged**